configure.ac

```diff
@@ -217,42 +199,89 @@ fi
 AC_SUBST([ENABLED_ASM])
 
 
-# FIPS
+# FIPS 140
 AC_ARG_ENABLE([fips],
     [AS_HELP_STRING([--enable-fips],[Enable FIPS 140-2, Will NOT work w/o FIPS license (default: disabled)])],
     [ENABLED_FIPS=$enableval],
     [ENABLED_FIPS="no"])
 
+if test "$ENABLED_FIPS" != "no"
+then
+    REPRODUCIBLE_BUILD_DEFAULT=yes
+fi
+
+# The FIPS options are:
+#   v5-RC8 - FIPS 140-3 (wolfCrypt WCv5.0-RC8)
```

Comment:
the careful qualification of the label is to avoid confusion.  the RC8 code snapshot is what went to the lab, but going forward we're expecting to update the lab with a new snapshot once we merge-and-tag, and that then will get a new name, v5-something-else, and v5 will become an alias for that.

Reply:
followup on this: will be pushing a commit shortly on this, message "configure.ac: for --enable-fips, make v5 an alias for v5-RC8 (alias to be updated after newer lab-approved snapshots are tagged), and add v5-ready and a placeholder v5-REL." and a second related commit, message "fips-check.sh: add linuxv5-ready (--enable-fips=v5-ready).".

I also pushed my branch in wolfSSL/fips#156 to github.com/wolfSSL/fips as branch `douzzer-linuxkm-fips-140-3`, used as the `FIPS_VERSION` in fips-check.sh for the `linuxv5-ready` test.  Better than having `FIPS_REPO` in github.com:douzzer/fips, but obviously I'll update this after the merge of wolfSSL/fips#156 .